llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -785,6 +785,7 @@ enum : unsigned {
   EF_AMDGPU_MACH_AMDGCN_GFX1200         = 0x048,
   EF_AMDGPU_MACH_AMDGCN_RESERVED_0X49   = 0x049,
   EF_AMDGPU_MACH_AMDGCN_GFX1151         = 0x04a,
+  EF_AMDGPU_MACH_AMDGCN_GFX1152         = 0x055,
```

Comment:
This table is supposed to be in ELF number order. Can you please move the new entry? Consider it pre-approved.